### DISCUSSION TITLE
I cannot run python script with pydantic from Git Bash for windows

### DISCUSSION THREAD

**Discussion by irishad** [pydantic/pydantic#2759] [Question]
I've installed pydantic as discribed in docs "pip install pydantic" and tested if pydantic is compiled:

import pydantic
print('compiled:', pydantic.compiled)

It works in Pycharm, but when I try to run this script from Git Bash for windows it doesn't work:
![Pydantic](https://user-images.githubusercontent.com/50150096/117495278-18198d80-af7e-11eb-8ec0-423ce7c112a6.png)

OS Windows 10 Pro

I'll be grateful for any help

**Comment by samuelcolvin** [comment]
This is not a problem with pydantic but with your python setup.

Looks like you are calling a different python executable in the command like that doesn't have pydantic in the python path.
